src/navigation-bar.tsx

```diff
@@ -581,7 +581,10 @@ export const NavigationBarAction: React.FC<NavigationBarActionProps> = ({childre
     const isInverse = useIsInverseVariant();
     const classes = useNavigationBarActionStyles({isInverse});
     return (
-        <Touchable {...touchableProps} className={classes.touchable}>
```

Comment:
I think that's intended, because `NavigationBarAction` tries to follow the specs regarding actions with icon in navbars.
I'd say if you don't want these styles, you'd better use a custom component built with a `Touchable` with your own styles, instead of using `NavigationBarAction`, wdyt?